.github/workflows/pull_request.yml

```diff
@@ -112,28 +112,28 @@ jobs:
       matrix:
         suite: [ timeout_modal, features, general, components ]
     needs: [python-dependencies, node-dependencies]
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     env:
       EQ_RUN_FUNCTIONAL_TESTS_HEADLESS: True
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Update to latest stable Chrome
         run: |
           wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
           sudo apt install ./google-chrome-stable_current_amd64.deb
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v3
         with:
-          node-version: "14.19.0"
+          node-version: "14"
```

Comment:
Hi, Just wanted to check
The earlier version had specific version(14.9.0), here we have changed it to Major Version (14). Any specific reason for this change?

Reply:
Hi, I have refactored this back to version 14.19.0. Cheers.